README.md

```diff
@@ -205,19 +207,23 @@ the regional metaweb [@Saravia2022Ecological]. Without clear documentation, it
 can be challenging to know if published probabilistic interaction webs describe
 local or regional interactions (@Tbl:prob provides examples of studies employing
 both types of probabilistic interaction networks), or if so-called probabilities
-are in reality *interaction scores* (i.e., non-probabilistic quantitative
-interactions analogous to interaction strengths). When probabilistic regional
+are in reality a form of interaction score (i.e., non-probabilistic quantitative
+interactions analogous to interaction strengths, which we won't cover further). 
+When probabilistic regional
 interactions are used and interpreted incorrectly as local interactions (and
 conversely), this may generate misleading findings during data analysis. We
 believe that a better understanding of the differences, similarities, and
 interconnections between these two probabilistic representations of ecological
 interactions would alleviate interpretation errors (e.g., when studying
-network-area relationships) and facilitate a more adequate utilization of
-interaction data.
+network-area relationships with metawebs or local webs) and facilitate a more 
+adequate use of interaction data.
+In the next sections, we therefore review the differences as we scale up from 
```

Comment:
L220: I think we should find a way to emphasize earlier in the text the three elements that become titled sections: pairwise interactions, local webs, metawebs. Either by putting them all in italics (pairwise interactions are not right now), or by mentioning the three sections to come in the paragraph just before (as I attempted here).